src/MeetingsSDKAdapter.test.js

```diff
@@ -157,11 +157,44 @@ describe('Meetings SDK Adapter', () => {
     });
   });
 
-  describe('muteVideoControl()', () => {
+  describe('handleLocalAudio()', () => {
+    beforeEach(() => {
+      meetingSDKAdapter.meetings[meetingID] = {
+        ...meeting,
+        localAudio: {
+          getAudioTracks: jest.fn(() => [{enabled: true}]),
+        },
+      };
+    });
+
+    test('mutes audio if the the audio track is enabled', () => {
+      meetingSDKAdapter.handleLocalAudio(meetingID);
+      expect(mockSDKMeeting.muteAudio).toBeCalled();
```

Comment:
Since `toBeCalled` is just an alias for `toHaveBeenCalled` and we're using `toHaveBeenCalled` elsewhere in this repo, can we standardize on using `toHaveBeenCalled`?

Reply:
sure